cli/command/run/cmd.go

```diff
@@ -116,3 +99,35 @@ func Run(cmd *cobra.Command, cli *command.SauceCtlCli, args []string) (int, erro
 
 	return exitCode, nil
 }
+
+// expandEnvMetadata expands environment variables inside metadata fields that are part of the job configuration.
```

Comment:
Can you add an example, e.g. if an env var is defined as CI_COMMIT_SHORT_SHA=foobar it replaces the config for `build: Release $CI_COMMIT_SHORT_SHA` to `build: Release foobar`